### DISCUSSION TITLE
개인 페이지 질문 입니다.

### DISCUSSION THREAD

**Discussion by Zerohertz** [awesome-devblog/awesome-devblog#791] [Q&A] [answered]
https://awesome-devblog.netlify.app/korean/people/Zerohertz

해당 링크에서 개인 정보가 조회되지 않는데,
시간이 지나야 생성되는 건가요?

그리고 혹시 한달 이전 글들은 피드에 원칙적으로 올라가지 않나요?

감사합니다.

**Comment by sarojaba** [comment, marked as answer]
개인 페이지는 현재 기능에 문제가 생겨 업데이트가 안되고 있습니다.
(언제 복구될지 잘 모르겠네요;;)

그리고 수집 시점에서 오래된 글(약 한달정도 이전)은 수집되지 않습니다.

**Reply by Zerohertz** [reply]
답변 감사드립니다!
